### PR TITLE
docs: change heading level for why updating often is easier

### PR DESCRIPTION
**Pull request by mschoettle**
## Changes

Moves the heading "Why updating often is easier, faster and safer" to level 2. It does not seem to belong to the section "Use the config:best-practices preset".

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/upgrade-best-practices.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Pinning your development dependencies means you, and your team, are using the sa
 This makes the developer-tool side of your builds reproducible.
 Debugging faulty versions of your tools is easier, because you can use Git to check out different versions of the tools.
 
-### Why updating often is easier, faster and safer
+## Why updating often is easier, faster and safer
```
Comment (rarkins) [inline docs/usage/upgrade-best-practices.md:91]: Looks like headings below this need to be moved from 4 to 3 too if so

**Review by HonkingGoose — COMMENTED** [review]
Good catch on the bad headings! 

Please also make the changes rarkins is suggesting in their comment. 🙂

File: docs/usage/upgrade-best-practices.md (hunk 1 above)
Comment (mschoettle) [reply docs/usage/upgrade-best-practices.md:91]: Doh! Absolutely

**Review by HonkingGoose — APPROVED** [review]
Good to go! 🥳

**Review by rarkins — APPROVED** [review]

**Pull request merged**